index.js

```diff
@@ -127,16 +148,23 @@ function buildRouteRate (pluginComponent, params, routeOptions) {
         if (typeof params.onExceeded === 'function') {
           params.onExceeded(req)
         }
+        let message = `Rate limit exceeded, retry in ${after}`
+        if (typeof params.errorMessage === 'string') {
+          message = params.errorMessage
+        } else if (typeof params.errorMessage === 'function') {
+          message = params.errorMessage()
+        }
 
         res.type('application/json').serializer(serializeError)
-        res.code(429)
+        res
+          .code(429)
           .header('X-RateLimit-Limit', params.max)
           .header('X-RateLimit-Remaining', 0)
           .header('Retry-After', params.timeWindow)
           .send({
             statusCode: 429,
             error: 'Too Many Requests',
-            message: `Rate limit exceeded, retry in ${after}`
+            message
```

Comment:
Have you changed the linting in this file? Would you mind removing the linting changes, and limit this to the actual change?